targets/TARGET_STM/README.md

```diff
@@ -377,6 +409,27 @@ You can change this in you local mbed_app.json:
 ```
 
 
+### Sleep feature
+
+ST MCUs feature several low-power modes, please check Reference Manual of each one for more details.
+
+- MBED sleep mode is usually mapped to ST SLEEP mode:
+  - CPU clock is off
+  - all peripherals can run and wake up the CPU when an interrupt or an event
+occurs
+
+- MBED deepsleep mode is mapped to ST STOP2 mode:
+  - all clocks in the VCORE domain are stopped
+  - the PLL, the MSI, the HSI and the HSE are disabled
+  - the LSI and the LSE can be kept running
+  - RTC can remain active
+
+Detailed sleep Mbed OS description : https://os.mbed.com/docs/mbed-os/latest/apis/power-management-sleep.html
+- debug profile is disabling deepsleep
+- deepsleep can also be disabled by application or drivers using sleep_manager_lock_deep_sleep()
+- deep-sleep-latency value is configured to 4 by default for STM32
```

Comment:
would it be worth documenting why 4 was selected here?

Reply:
https://github.com/ARMmbed/mbed-os/pull/11767

@LMESTM